src/registry/MergedRegistry.ts

```diff
@@ -93,6 +93,16 @@ export class MergedRegistry implements IRegistry {
     );
   }
 
+  async getWarpRoute(id: WarpRouteId): Promise<WarpCoreConfig | null> {
+    const results = await this.multiRegistryRead((r) => r.getWarpRoute(id));
+    return results.find((r) => !!r) || null;
+  }
+
+  async getWarpRoutes(filter?: WarpRouteFilterParams): Promise<WarpRouteConfigMap> {
```

Comment:
what if I want to specify multiple symbols or chains?

Reply:
Call again I guess? I'm open to alternative filter designs but didn't want to over-engineer this. 
One though could be a list of filters but some people may expect them the intersection of the filter results, not the union

Reply:
Added info to the function comment header to clarify

Reply:
okay yeah no need to complicate now